doc/tools/coqrst/coqdoc/main.py

```diff
@@ -32,10 +33,17 @@
 COQDOC_SYMBOLS = ["->", "<-", "<->", "=>", "<=", ">=", "<>", "~", "/\\", "\\/", "|-", "*", "forall", "exists"]
 COQDOC_HEADER = "".join("(** remove printing {} *)".format(s) for s in COQDOC_SYMBOLS)
 
+def tempfile():
+    fd, filename = mkstemp(prefix="coqdoc-", suffix=".v")
+    if platform.system().startswith("CYGWIN"):
+        # coqdoc doesn't accept cygwin style paths in the form "/cygdrive/c/..."
+        filename = check_output(["cygpath", "-w", filename]).strip().decode("utf-8")
+    return fd, filename
+
 def coqdoc(coq_code, coqdoc_bin=None):
     """Get the output of coqdoc on coq_code."""
     coqdoc_bin = coqdoc_bin or os.path.join(os.getenv("COQBIN"), "coqdoc")
-    fd, filename = mkstemp(prefix="coqdoc-", suffix=".v")
+    fd, filename = tempfile()
```

Comment:
I think tempfile should take the same arguments as mkstemp.

Reply:
Or just inline the code instead of having a separate function--my original tempfile() was more complex.